### PR TITLE
WIP: [LPT] Migration to nGraph

### PR DESCRIPTION
**Pull request by eshoguli**

### CODE REVIEW

File: inference-engine/src/low_precision_transformations/include/low_precision_transformations/layer_transformation.hpp [hunk 1]
```diff
@@ -27,8 +27,8 @@
  *    dequantization layers printing
  *
  *****************************************************/
-// #define LPT_ORIGINAL_MODEL_PATH "C:\\Projects\\temp\\original"
-// #define LPT_TRANSFORMED_MODEL_PATH "C:\\Projects\\temp\\transformed"
+//#define LPT_ORIGINAL_MODEL_PATH "/localdisk/slyalin/orig.model"
```
Comment (ilyachur) [inline inference-engine/src/low_precision_transformations/include/low_precision_transformations/layer_transformation.hpp:30]: Please remove these comments!

File: inference-engine/include/details/ie_no_release.hpp [hunk 2]
```diff
@@ -4,14 +4,17 @@
...
- * 
+ *
  * @file ie_no_release.hpp
  */
 #pragma once
 
 namespace InferenceEngine {
 namespace details {
 
+// TODO: eshoguli: remove
```
Comment (ilyachur) [inline inference-engine/include/details/ie_no_release.hpp:15]: Do we really need such TODO?

File: inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp [hunk 3]
```diff
@@ -802,6 +805,7 @@ std::shared_ptr<CNNNetworkImpl> convertFunctionToICNNNetwork(const std::shared_p
                 ptr.reset(new Data(outName,
                                    {details::convertPrecision(layer->get_output_element_type(i)), dims,
                                     TensorDesc::getLayoutByDims(dims)}));
+                // std::cerr << "Set precision " << layer->get_output_element_type(i).get_type_name() << " for layer " << layer->get_friendly_name() << "\n";
```
Comment (ilyachur) [inline inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp:808]: Redundant comments.

**Comment by dmitry-gorokhov** [comment]
@eshoguli Is the PR still actual? Please close if not.

**Comment by dmitry-gorokhov** [comment]
Closing as not actual anymore

**Pull request closed without merging**